docs/en/logs/logs-ui-intro.asciidoc

```diff
@@ -2,16 +2,16 @@
 [role="xpack"]
 == {logs-app}
 
-After you have set up logs ingestion and log data is streaming to {es}, you can view real-time and historic logs in a compact, customizable display.
+After setting up logs ingestion and log data is streaming to {es}, you can view real-time and historical logs in a compact, customizable display.
 The log data is correlated with metrics data in the {metrics-guide}/infrastructure-ui-overview.html[{metrics-app}], making it easier for you to diagnose problems.
 You can also view related application traces or uptime information where available.
 
-You can stream the logs in real time, or view historical logs from a specified time range.
+You can stream the logs in real-time, or view historical logs from a specified time range.
```

Comment:
I would not hyphenate.

The rule I've always heard is that real-time is the adjective form of real time (the noun). :-)

Reply:
Thanks @dedemorton! Good to know.